project_stock/models/account_analytic_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    task_id = fields.Many2one(comodel_name="project.task", string="Project Task",)
```

Comment:
The other name already exists

```suggestion
    stock_task_id = fields.Many2one(comodel_name="project.task", string="Project Task",)
```